config_arch

```diff
@@ -19,6 +19,7 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+scriptdir="$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)"
```

Comment:
`config_arch` is a script. So, it is the directory where this script is located. Basically, this change allows us to call `config_arch` from anywhere.

Reply:
I would still call it sourcedir or similarly, since you are using it below to pass the include paths to gcc

Reply:
A nitpick. I don't like the name `sourcedir` because it misleads me to think about the `src` folder in GDRCopy. I propose that we use `topdir` instead.